packages/cli/index.js

```diff
@@ -1,3 +1,9 @@
+/* istanbul ignore if */
+if (parseInt(process.version.split('.')[0].substring(1), 10) <= 11) {
```

Comment:
Should we add a `semver` dep and import `engines.node` from the package.json?

```suggestion
if (!semver.satisfies(process.version, pkg.engines.node)) {
```
<br>

If not, we should at least assert against `12` so it's easy to search for and update

```suggestion
if (parseInt(process.version.split('.')[0].substring(1), 10) < 12) {
```